docs/source/explore/get-started/vr-controls.rst

```diff
@@ -15,15 +15,114 @@ Use these controls on your VR controllers so that you can enjoy your experience
 
```

Comment:
Since we're focusing less on HMD and more on Desktop, we should remove any reference to "VR equipment is better than Desktop". So change the first paragraph to something more vague like 

```suggestion
To interact with people in 3D and track body movements, you will need to use VR equipment such as the Oculus Rift or HTC Vive. With VR equipment, you can have a fully immersive experience in the 3D metaverse.
```

Reply:
Should we use the word metaverse? I feel like we're shifting away from what it means to a lot of users. Can we replace that with just High Fidelity instead?

Reply:
I modified this line and made it High Fidelity instead.